tests/python/gaia-ui-tests/gaiatest/apps/system/regions/time_picker.py

```diff
@@ -16,54 +17,44 @@ class TimePicker(Base):
     _done_button_locator = (By.CSS_SELECTOR,
                             '.value-selector-time-picker-buttons > button.value-selector-confirm')
 
-    _hour_picker_locator = (By.CSS_SELECTOR, '.value-picker-hours > div')
-    _minutes_picker_locator = (By.CSS_SELECTOR, '.value-picker-minutes > div')
-    _hour24_picker_locator = (By.CSS_SELECTOR, '.value-picker-hour24-state > div')
+    _hour_picker_locator = (By.CLASS_NAME, 'value-picker-hours')
+    _minutes_picker_locator = (By.CLASS_NAME, 'value-picker-minutes')
+    _hour24_picker_locator = (By.CLASS_NAME, 'value-picker-hour24-state')
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
         self.marionette.switch_to_frame()
-        self.wait_for_element_displayed(*self._current_element(*self._hour_picker_locator))
+        Wait(self.marionette).until(expected.element_displayed(*self._hour_picker_locator))
 
-        # TODO: wait for the time piker to fade in Bug 1038186
+        # TODO: wait for the time picker to fade in Bug 1038186
         time.sleep(2)
 
     def tap_done(self):
         self.marionette.find_element(*self._done_button_locator).tap()
         self.wait_for_element_not_displayed(*self._time_picker_locator)
-        # TODO: wait for the time piker to fade out Bug 1038186
+        # TODO: wait for the time picker to fade out Bug 1038186
         time.sleep(2)
         self.apps.switch_to_displayed_app()
 
     @property
     def hour(self):
-        return self.marionette.find_element(*self._current_element(*self._hour_picker_locator)).text
-
+        return self._current(self._hour_picker_locator).text
+         
     @property
     def minute(self):
-        return self.marionette.find_element(*self._current_element(*self._minutes_picker_locator)).text
+       return self._current(self._minutes_picker_locator).text
 
     def spin_minute(self):
```

Comment:
For consistency with the master branch, I would rename this function `add_minute` like [done here](https://github.com/mozilla-b2g/gaia/commit/59f71e42d4137f194ca50709497f6e3706dad10a#diff-3ec38c276967a961e2f4c4ac0c8fe0f9R53)